x/concentrated-liquidity/types/msgs_test.go

```diff
@@ -160,6 +173,103 @@ func TestMsgCreatePosition(t *testing.T) {
 	}
 }
 
+func TestMsgAddToPosition(t *testing.T) {
+	tests := []struct {
+		name       string
+		msg        types.MsgAddToPosition
+		expectPass bool
+	}{
+		{
+			name: "proper msg",
+			msg: types.MsgAddToPosition{
+				PositionId:      1,
+				Sender:          addr1,
+				Amount0:         sdk.OneInt(),
+				Amount1:         sdk.OneInt(),
+				TokenMinAmount0: sdk.OneInt(),
+				TokenMinAmount1: sdk.OneInt(),
+			},
```

Comment:
nit: consider defining a default message and changing the necessary fields via callback functions to reduce code duplication

Reply:
+1, would make tests more concise & readable but non-blocking